src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/DeploymentOptions.cs

```diff
@@ -207,6 +207,18 @@ public class DeploymentOptions
         public DeploymentOptions()
         {
             DacDeployOptions options = new DacDeployOptions();
+
+            // Adding these defaults to ensure behavior similarity with other tools.
```

Comment:
add that the dacfx wizard and ssms import/export wizards use these defaults

Reply:
Hey @kisantia, you mean in comment? haha I had that and then deleted :P 
Will add comment.

Reply:
yup I meant in the comment :)